webhook_to_fedora_messaging/models/util.py

```diff
@@ -7,21 +7,16 @@
 from sqlalchemy.ext.compiler import compiles
 from sqlalchemy.sql.expression import FunctionElement
 from sqlalchemy.types import DateTime as SQLDateTime
+from datetime import datetime, UTC
 
 from uuid import uuid4
 
 
-class utcnow(FunctionElement):
+def utcnow():
     """
     Current timestamp in UTC for SQL expressions
     """
-    type = SQLDateTime
-    inherit_cache = True
-
-
-@compiles(utcnow, "postgresql")
-def _postgresql_utcnow(element, compiler, **kwargs):
-    return "(NOW() AT TIME ZONE 'utc')"
+    return str(datetime.now(UTC))
```

Comment:
I don't think the postgresql driver will be very happy if we give it a string instead of a datetime. The SQLite driver will probably not mind though, because it's storing datetimes as strings anyway.

As a result, the `utcnow` function will probably boil down to `functools.partial(datetime.now, tz=UTC)`. But you can keep it as a function either, I don't have reasons to think one is better than the other :-)

Reply:
I did not completely understand the request here. Should I revert the `_postgresql_utcnow` function for use with the Postgres compiler and keep the string converted `datetime.now(UTC)` object for Sqlite cases? I removed the `utcnow()` function for now for a oneliner string converted `functools.partial(datetime.now, tz=UTC)` but I am 100% sure that would not satiate the Postgres compiler.